scripts/clean-instance.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+#
+# This file is part of Invenio.
+# Copyright (C) 2015, 2016, 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+# check environment variables:
+if [ -v ${INVENIO_WEB_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_WEB_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_WEB_HOST=192.168.50.10"
```

Comment:
Should we add the default ones if one is not set ?

Reply:
Yes, we could.  Currently, there is `.inveniorc` that you can do `source .inveniorc` and then you have all the variables set.  In which case you don't even have to set default ones...

I'd say (1) let's have a look at the current `.inveniorc` content, to see whether the values suit your habits, or whether we amend them in some way; and then let's agree that (2a) either we shall be doing `source .inveniorc` everytime, or (3b) we'd put the values into scripts for safety.  This should also be OK to do, since the scripts were "cloned" from generic Invenio ones to custom COD ones; it would be simply a bit harder to port changes over.  Would you prefer 3a or 3b on the East End? We can again see IRL...